ext/opentelemetry-ext-docker-tests/tests/postgres/test_psycopg_functional.py

```diff
@@ -0,0 +1,130 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import unittest
+
+import psycopg2
+
+from opentelemetry import trace as trace_api
+from opentelemetry.ext.psycopg2 import trace_integration
+from opentelemetry.sdk.trace import Tracer, TracerProvider
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+    InMemorySpanExporter,
+)
+from pytest import fixture
+
+POSTGRES_HOST = os.getenv("POSTGRESQL_HOST ", "localhost")
+POSTGRES_PORT = int(os.getenv("POSTGRESQL_PORT ", "5432"))
+POSTGRES_DB_NAME = os.getenv("POSTGRESQL_DB_NAME ", "opentelemetry-tests")
+POSTGRES_PASSWORD = os.getenv("POSTGRESQL_HOST ", "testpassword")
+POSTGRES_USER = os.getenv("POSTGRESQL_HOST ", "testuser")
+
+
+class TestFunctionalPsycopg(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._connection = None
+        cls._cursor = None
+        cls._tracer_provider = TracerProvider()
+        cls._tracer = Tracer(cls._tracer_provider, None)
+        cls._span_exporter = InMemorySpanExporter()
+        cls._span_processor = SimpleExportSpanProcessor(cls._span_exporter)
+        cls._tracer_provider.add_span_processor(cls._span_processor)
+        trace_integration(cls._tracer)
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls._cursor:
+            cls._cursor.close()
+        if cls._connection:
+            cls._connection.close()
+
+    @fixture(autouse=True)
+    def connect_to_db(self):
+        if self._connection is None:
+            # Try to connect to DB
+            for i in range(5):
+                try:
+                    self._connection = psycopg2.connect(
+                        dbname=POSTGRES_DB_NAME,
+                        user=POSTGRES_USER,
+                        password=POSTGRES_PASSWORD,
+                        host=POSTGRES_HOST,
+                        port=POSTGRES_PORT,
+                    )
+                    self._cursor = self._connection.cursor()
+                    break
+                except Exception as err:
+                    print(err)
+                time.sleep(5)
+            else:
+                raise Exception("Failed to connect to DB")
+
+    def setUp(self):
+        self._span_exporter.clear()
+
+    def validate_spans(self):
+        spans = self._span_exporter.get_finished_spans()
+        self.assertEqual(len(spans), 2)
+        for span in spans:
+            if span.name == "rootSpan":
+                root_span = span
+            else:
+                child_span = span
+            self.assertIsInstance(span.start_time, int)
+            self.assertIsInstance(span.end_time, int)
+        self.assertIsNot(root_span, None)
```

Comment:
why use both assertIsNotNone below and assertIsNot(x, None)? Seems like we could just standardize on a single one.